master/internal/trial.go

```diff
@@ -235,6 +242,25 @@ func (t *trial) maybeAllocateTask(ctx *actor.Context) error {
 	return nil
 }
 
+const (
+	// InvalidHPKillDelay the delay before we forcibly kill a trial that said it had an invalid HP.
+	InvalidHPKillDelay = 10 * time.Second
```

Comment:
can we lower this to 3 seconds.  that matches the grace period we give the launched processes in the harness